airflow-core/src/airflow/ui/public/i18n/locales/ko/common.json

```
@@ -90,6 +90,7 @@
     "back": "뒤로",
     "defaultMessage": "예기치 않은 오류가 발생했습니다.",
     "home": "홈",
+    "invalidUrl": "페이지를 찾을 수 없습니다. URL을 확인하고 다시 시도해주세요.",
```

[nitpick] it'd be better with the space 'v'
```suggestion
    "invalidUrl": "페이지를 찾을 수 없습니다. URL을 확인하고 다시 시도해 주세요.",
```

---

LGTM